core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -185,8 +200,11 @@ private void forceDrawIndicatorAtPosition(int position) {
         invalidate();
     }
 
+    public void detach(ViewPager viewPager) {
```

Comment:
I don't see a matching attach method. what was the design choice for this?

How come you pass the viewpager when there is a viewpager as a field?

Could you use `onDetachedFromWindow` to detach, instead of allowing the client to decide when to detach?